# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1471676654 -7200
#      Sat Aug 20 09:04:14 2016 +0200
# Node ID 2e49ab3d6b481a4747f683bfed4b53ec5db0b001
# Parent  752d79f867817d68f68f6b75118077749305e8fa
ObjectProxy.cc: Prevent calling slotUpdate on uninitialized object (#48519)

diff --git a/libgui/graphics/ObjectProxy.cc b/libgui/graphics/ObjectProxy.cc
--- a/libgui/graphics/ObjectProxy.cc
+++ b/libgui/graphics/ObjectProxy.cc
@@ -81,17 +81,17 @@ ObjectProxy::setObject (Object* obj)
   init (obj);
 }
 
 void
 ObjectProxy::update (int pId)
 {
   if (octave_thread::is_octave_thread ())
     emit sendUpdate (pId);
-  else
+  else if (m_object)
     m_object->slotUpdate (pId);
 }
 
 void
 ObjectProxy::finalize (void)
 {
   emit sendFinalize ();
   init (0);
